extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -722,18 +738,50 @@ export class Bind {
     });
   }
 
+  /**
+   * Scans the element for bindings and adds up to |quota| to `outBindings`.
+   * Also updates ivars `boundElements_` and `expressionToElements_`.
+   * @param {!Element} element
+   * @param {number} quota
+   * @param {!Array<!BindBindingDef>} outBindings
+   * @return {boolean} Returns true if `element` contains more than `quota`
+   *     bindings. Otherwise, returns false.
+   */
+  scanElement_(element, quota, outBindings) {
+    let quotaExceeded = false;
+    let boundProperties = this.boundPropertiesInElement_(element);
+    // Stop scanning once |limit| bindings are reached.
+    if (boundProperties.length > quota) {
+      boundProperties = boundProperties.slice(0, quota);
```

Comment:
Nit: we could turn `boundProperties` into a constant if we change this to `boundProperties.length = quota`

Reply:
Good idea.